server/index.js

```diff
@@ -1,20 +1,58 @@
 const express = require('express');
 const path = require('path');
 const axios = require('axios');
+const { join } = require('path');
```

Comment:
It looks like you import path on line 2. Any reason just to restructure join here? Or, do we not need the path import anymore?

Reply:
was just sloppy. refactored.